content/yaml/testing.md

```diff
@@ -50,5 +50,36 @@ scripts:
 
 ## Flutter drive test
 
-    flutter emulators --launch apple_ios_simulator                  # for android use: flutter emulators --launch emulator
-    flutter drive --target=test_driver/my_drive_target.dart
+#### On a mobile emulator
```

Comment:
I'd rephrase as: Running on a mobile emulator or simulator
This heading is currently ####, but should be ### for consistency.

Reply:
Fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/146/commits/0c31b30a4a008e8d6f708454a8033844adfcb3a3.

Just to stress the point that you can't run web app tests on ios/android emulator, or that you can't run android app in a browser, said it that way.